header/header.go

```diff
@@ -120,6 +120,9 @@ func (eh *ExtendedHeader) Validate() error {
 			eh.RawHeader.Version.App)
 	}
 
+	if eh.Commit == nil {
+		return fmt.Errorf("nil commit at height %d", eh.Height())
+	}
```

Comment:
Should be inside core part of ValidateBasic. Let submit fix there

Reply:
Yea actually